includes/payments/class-wc-woomercadopago-payment-abstract.php

```diff
@@ -1542,6 +1561,54 @@ public function field_checkout_support_problem() {
 		);
 	}
 
+		/**
+	 * Field Installments and Interest Configuration Title
+	 *
+	 * @return array
+	 */
+	public function field_mp_psj_title() {
+		$message_support_title = __( 'Set up your installment and interest payments', 'woocommerce-mercadopago' );
+
+		return array(
+			'title' => $message_support_title,
+			'type'  => 'title',
+			'class' => 'mp_subtitle_bd_mb mp-mg-0',
+		);
+	}
+
+	/**
+	 * Field Installments and Interest Configuration Description
+	 *
+	 * @return array
+	 */
+	public function field_mp_psj_description() {
+		$message_support_description = __( 'At Mercado Pago you can choose the fee you pay for each purchase and also offer interest-free installments to your customer.', 'woocommerce-mercadopago' );
+
+		return array(
+			'title' => $message_support_description,
+			'type'  => 'title',
+			'class' => 'mp_small_text',
+		);
+	}
+
+	/**
+	 * Field Checkout Support Description Link
+	 *
+	 * @return array
+	 */
+	public function field_mp_psj_description_link() {
+		$message_link = __( 'Set up installment and interest', 'woocommerce-mercadopago' );
+
+		return array(
+			'title' => sprintf(
+				'%s',
+				'<a href="' . $this->get_country_link_mp( $this->checkout_country ) . 'costs-section#from-section=menu" target="_blank">' . $message_link . '</a>'
```

Comment:
Do jeito que está sendo feito não precisaria chamar o sprintf, já está montando tudo no parâmetro.

para usar sprintf seria interessante montar a string com os placeholder como:

```php
sprintf(
    '<a href="%scosts-section#from-section=menu" target="_blank">%s</a>',
    $this->get_country_link_mp( $this->checkout_country ),
    $message_link 
);
```

Ou deixar o title como:

`'title' => '<a href="' . $this->get_country_link_mp( $this->checkout_country ) . 'costs-section#from-section=menu" target="_blank">' . $message_link . '</a>'`

Reply:
Opaa, verdade! Obrigada, Lira!